tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -195,7 +195,7 @@ const waitBetweenFileUploadOperations = async function () {
   timeOfLastUploadOperation = Date.now()
 }
 
-Given('the following files have been deleted by user {string}', async function (user, table) {
+Given('the following files/folders/elements have been deleted by user {string}', async function (user, table) {
```

Comment:
Just to be consistent.
```suggestion
Given('the following files/folders/resources have been deleted by user {string}', async function (user, table) {
```